app/src/main/java/org/mozilla/rocket/content/CouponAdapter.kt

```diff
@@ -37,10 +37,13 @@ class CouponAdapter(private val listener: ContentPortalListener) : ListAdapter<C
     override fun onBindViewHolder(holder: CouponViewHolder, position: Int) {
         val item = getItem(position) ?: return
         holder.bind(item, View.OnClickListener {
-            TelemetryWrapper.clickOnEcItem(
+            TelemetryWrapper.clickOnPromoItem(
                     pos = position.toString(),
+                    id = item.id,
+                    feed = item.feed,
                     source = item.link.source,
-                    category = item.link.name
+                    category = item.category,
```

Comment:
now link's name is duplicate with category.
we can't change the schema easily, I'd like to prevent this from happening.
can we re-use the name from category's getter in Coupon class ?